Fix family pages

araiford opened the pull request:
**Significant changes:**

- Adds a content-wrapper class which facilitates max-width and horizontal centering for content on the helix site.
- Creates a new component-row.html which gets included in each family page, rather than having duplicate raw html on each one.

**Minor changes:**

- Adjusts the component detail header so that the breadcrumbs and buttons are on the same horizontal line.

nimbinatus commented on assets/css/_sass/website/_docs-page.scss:
```diff
@@ -54,9 +58,8 @@
-    margin: 0;
+  .content-wrapper {
+
     ul#markdown-toc {
```
I don't remember defining this multiple times, but this class/identifier is named both here and in _family-page.scss (and actually under com-code above, too). We should only define it once as it seems to be under the same parent class (though maybe it's `.component-wrapper` > `.component-wrapper` > `ul#markdown-toc` here?). I thought I had it in a different parent class for the family page... As it stands, though, the page links are now cyan rather than gray on the family pages, which doesn't match the design spec I had. Is that intentional?

technabors replied:
FWIW I prefer the new styling proposed in this PR.

araiford replied:
Agree -- I saw that the <article> element had css rules in several places and was unsure why.  I'll clean that up and make sure that .content-wrapper is defined in one place.  The family page component-row links becoming cyan was not intentional, but I actually like it that way now that I've seen the difference.  I'm going to consult Ty on it.

nimbinatus commented on assets/css/_sass/website/_docs-page.scss:
```diff
@@ -54,9 +58,8 @@
-  article {
-    padding: 0 28px;
-    margin: 0;
+  .content-wrapper {
```
Is this `.content-wrapper` inside of a `.content-wrapper`? Was that intentional?

araiford replied:
ah -- nope, the previous structure was:

.docs-content {
  article {}
}

this was an error on my part.  I should have changed the second, nested .content-wrapper to .docs-content.

Will change

nimbinatus commented on templates/component.html:
```diff
@@ -2,34 +2,39 @@
-      <div class="hxRow">
-        <div class="hxCol">
-          <div class="component-title">{{page.title}}</div>
+    <div class="content-wrapper">
```
Did you also make these changes to the component-code template, or are we skipping that page for now? I don't see it in this PR, but maybe it happened in a separate one?

araiford replied:
I didn't.  We need to figure out the relationship between the Helix website and the helix-ui site.  So, for right now, I'm holding off on the component code pages.

araiford commented:
@nimbinatus 

- Removed redundant, nested rules for .content-wrapper.
- Removed some nesting around the #markdown-toc rule, since it's using ID to style.
- Removed the anchor tag styling within the #markdown-toc rules.  Ty's design file has those links as cyan, so this change makes them more like Ty's spec.

I also removed a couple chunks of sass that styled H* tags in the component  and docs pages.  In looking through the newer docs that have come out of the sprint, I think we need to update the contribution template file to use H* tags slightly differently.  Ty's design file has a larger gap between headers.  AKA  where we are using H2s and H3s in the docs, we should probably be using H3s and H5s.  Nothing major though, and we're at a good point to change that.  We can do that in a separate branch though since this one is already pretty bloated.

nimbinatus commented:
Ah, so there's H* guidance in the design file? Good to know for next time. I was mainly trying to standardize on a simple model to get us started. Let's open up a new PR on the heading standardization as you suggested.
